couchbase/tests/test_couchbase.py

```diff
@@ -193,8 +197,26 @@ def _assert_bucket_metrics(aggregator, tags, device=None):
 
 def _assert_stats(aggregator, node_tags, device=None):
     for mname in NODE_STATS:
-        aggregator.assert_metric('couchbase.by_node.{}'.format(mname), tags=node_tags, count=1, device=device)
+        # These 2 metrics are available ~30 seconds post start. The integrations test won't see it, but the e2e will
+        if mname == 'index_data_size' or mname == 'index_disk_size':
+            aggregator.assert_metric('couchbase.by_node.{}'.format(mname), tags=node_tags, device=device, at_least=0)
```

Comment:
If timing is definitely the issue with these metrics, then you could maybe first try asserting the other metrics, then `sleep()` for a few seconds, and then try asserting these metrics